src/tests/functional/inference_engine/transformations/offline_transformations/pruning_test.cpp

```diff
@@ -4684,4 +4684,561 @@ TEST(TransformationTests, CheckReshapeWithNoConstInShape) {
+        auto weights3 = create_constant_with_zeros({8, 6, 1, 1}, {{1, 2}, {}, {}, {}});
+        auto conv3 = std::make_shared<opset5::Convolution>(split->output(1), weights3, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        auto weights4 = create_constant_with_zeros({8, 8, 1, 1}, {{1, 2}, {}, {}, {}});
+        auto conv4 = std::make_shared<opset5::Convolution>(split->output(2), weights4, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        function = std::make_shared<ngraph::Function>(OutputVector{conv2, conv3, conv4}, ParameterVector{input});
+    }
+
+    {
+        auto input = std::make_shared<opset5::Parameter>(element::f32, PartialShape{1, 3, 8, 8});
+
+        auto weights1 = opset5::Constant::create(element::f32, {10, 3, 1, 1}, {1});
+        auto conv1 = std::make_shared<opset5::Convolution>(input, weights1, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        auto split_lengths = std::make_shared<opset5::Subtract>(opset5::Constant::create(element::i32, Shape{3}, {2, -1, 8}),
+                                                           opset5::Constant::create(element::i32, Shape{3}, {1, -5, 3}));
+        auto split = std::make_shared<opset5::VariadicSplit>(conv1, opset5::Constant::create(element::i32, {}, {1}),
+                                                             split_lengths);
+        auto weights2 = create_constant_with_zeros({8, 1, 1, 1}, {{1, 2}, {}, {}, {}});
+        auto conv2 = std::make_shared<opset5::Convolution>(split->output(0), weights2, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        auto weights3 = create_constant_with_zeros({8, 4, 1, 1}, {{1, 2}, {}, {}, {}});
+        auto conv3 = std::make_shared<opset5::Convolution>(split->output(1), weights3, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        auto weights4 = create_constant_with_zeros({8, 5, 1, 1}, {{1, 2}, {}, {}, {}});
+        auto conv4 = std::make_shared<opset5::Convolution>(split->output(2), weights4, Strides{1, 1},
+                                                           CoordinateDiff{0, 0}, CoordinateDiff{0, 0}, Strides{1, 1});
+        function_ref = std::make_shared<ngraph::Function>(NodeVector{conv2, conv3, conv4}, ParameterVector{input});
+    }
+
+    manager.register_pass<pass::Pruning>();
+    comparator.enable(FunctionsComparator::CmpValues::CONST_VALUES);
+    comparator.enable(FunctionsComparator::CmpValues::ATTRIBUTES);
+    comparator.enable(FunctionsComparator::CmpValues::ACCURACY);
```

Comment:
Please call pruning transformation in your tests :)
```suggestion
    comparator.enable(FunctionsComparator::CmpValues::ACCURACY);
     manager.run_passes(function);
```

Reply:
> Please call pruning transformation in your tests :)

Calling manager.run_passes(function); from here is not correct. This transformation will be called in TransformationTestsF::TearDown method.